android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -361,6 +369,10 @@ public void onDrawerStateChanged(int newState) {
             ab.setDisplayHomeAsUpEnabled(true);
             ab.setDisplayHomeAsUpEnabled(true);
             smoothActionBarToggle.syncState();
+        } else if(toolbar!=null) {
+            if(toolbar.getTitle().equals(getString(R.string.menu_about))) {
```

Comment:
Change to `toolbar!=null && toolbar.getTitle().equals(getString(R.string.menu_about))`